show/chassis_modules.py

```diff
@@ -102,3 +108,55 @@ def midplane_status(chassis_module_name):
         click.echo(tabulate(table, header, tablefmt='simple', stralign='right'))
     else:
         click.echo('No data available in CHASSIS_MIDPLANE_TABLE\n')
+
+@chassis.command()
+@click.argument('systemportname', required=False)
+@click.option('--namespace', '-n', 'namespace', required=True if multi_asic.is_multi_asic() else False, 
+                default=None, type=str, show_default=False, help='Namespace name or all')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def system_ports(systemportname, namespace, verbose):
+    """Show VOQ system ports information"""
+
+    cmd = "voqutil -c system_ports"
+
+    if systemportname is not None:
+        cmd += " -i \"{}\"".format(systemportname)
+    
+    if namespace is not None:
+        cmd += " -n {}".format(namespace)
+
+    clicommon.run_command(cmd, display_cmd=verbose)
+
+@chassis.command()
+@click.argument('ipaddress', required=False)
+@click.option('--asicname', '-x', 'asicname', default=None, type=str, show_default=False, help='Asic name')
```

Comment:
why not use `-n` here ? similar to `show chassis system_ports`

Reply:
Please see my response to comments on line# 146-149

Reply:
Changed "asicname" option selector to "-n" as suggested